src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -347,37 +347,40 @@ protected bool BuildInternal (string projectOrSolution, string target, string []
 			var start = DateTime.UtcNow;
 			var args  = new StringBuilder ();
 			var psi   = new ProcessStartInfo (XABuildExe);
+			var responseFile = Path.Combine (XABuildPaths.TestOutputDirectory, Path.GetDirectoryName (projectOrSolution), "msbuild.rsp");
```

Comment:
How is this *used*?  Or is `msbuild.rsp` "special" and MSBuild automatically reads it?

The [docs](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-response-files?view=vs-2017#msbuildrsp) are not entirely clear:

> This file, *MSBuild.rsp*, must be in the same directory as *MSBuild.exe*, otherwise it will not be found. 

We almost certainly are not writing `msbuild.rsp` into the same directory as `MSBuild.exe`...

Reply:
That is different from the docs from `msbuild /?`

```
@<file>            Insert command-line settings from a text file. To specify
                     multiple response files, specify each response file
                     separately.

                     Any response files named "msbuild.rsp" are automatically
                     consumed from the following locations:
                     (1) the directory of msbuild.exe
                     (2) the directory of the first project or solution built
```

Reply:
Its also different from the behaviour I have seen from actual MSBuild.